src/webapp/api.py

```diff
@@ -135,6 +134,29 @@ def handle_classify(json_data):
 
 @socketio.on("endGame")
 def handle_endGame(json_data):
-    # TODO: implement me!
+    """
+        Event which ends the final game of the two players. The players provide
+        their scores and the player with the highest score is deemed the winner.
+        The two scores are finally stored in the database.
+    """
+    date = datetime.datetime.today()
     data = json.loads(json_data)
-    emit("endGame", data)
+    # Get data from given player
+    game_id = data["gameId"]
+    score_player = data["score"]
+    player_id = data["playerId"]
+    name_player = data["name"]
+    # Insert score information into db
+    models.insert_into_scores(name_player, score_player, date)
+    # Create a list containing player data which is sent out to both players
+    return_data = {
```

Comment:
hvorfor returnerer du dette? skal de ikke få vite om de har vunnet eller ikke?

Reply:
hvordan er flyeten i multiplayer da? I single så blir haswon sendt tilbake fra classify. alt endGame gjør i single player er å rydde i db og lagre highscore. hvorfor må den reutrnere noe som helst i mulitplaiyer?

Reply:
fordi clientene ikke snakker sammen og de vil vite hvem av de to som vant

Reply:
Jeg kan endre det sånn at kun vinneren sin player ID blir returnert

Reply:
gjør det frontend vil